### DISCUSSION TITLE
magick convert ... sixel:- ?

### DISCUSSION THREAD

**Discussion by Nevrdid** [3rd/image.nvim#144] [General]
Hi,
I maybe open that discussion for nothing but as i'm currently starting a new neorg setup from scratch i would like to see my pictures so here i am x')
I basicly just would like to know if you know that magick handle well the sixel conversion so, is it really that hard to add the feature ? I mean, even if the magick rock don't implement it, can't we execute external magick and catch the output or something like that ?
I already fixed it this way for my nnn preview script however that was only a shell script, i don't think i'm able to help you more than droping the idea here x)
Anyway, thank for your work but as i will not change my foot+tmux setup, i can't really give any other feedback.
Have a nice day.

**Comment by 3rd** [comment]
Hey, it's less of an image processing issue and more of a rendering issue.
We're investigating using sixels, not sure yet how positioning works, or if it's able to render images on top of stuff and not clear them / parts of them if an underlying cell gets updated as that would cause issues.
Will update all the sixel related issues once we know.
